test/01_cond_field/process_simple.py

```diff
@@ -98,29 +95,33 @@ def process(self):
         # central_moments_quantity = moments(root_quantity, moments_fn=central_moments, mom_at_bottom=True)
         # central_moments_mean = estimate_mean(central_moments_quantity)
 
-        #self.process_target_var(estimator)
+        # self.process_target_var(estimator)
         self.construct_density(estimator, tol=1e-8)
 
     def process_target_var(self, estimator):
         n0, nL = 100, 3
         n_samples = np.round(np.exp2(np.linspace(np.log2(n0), np.log2(nL), self.n_levels))).astype(int)
 
-        n_estimated = estimator.bs_target_var_n_estimated(target_var=1e-5, sample_vec=n_samples)  # number of estimated sampels for given target variance
+        n_estimated = estimator.bs_target_var_n_estimated(target_var=1e-5,
+                                                          sample_vec=n_samples)  # number of estimated sampels for given target variance
         estimator.plot_variances(sample_vec=n_estimated)
         estimator.plot_bs_var_log(sample_vec=n_estimated)
 
     def construct_density(self, estimator, tol=1.95, reg_param=0.0):
         """
         Construct approximation of the density using given moment functions.
-        Args:
-            moments_fn: Moments object, determines also domain and n_moments.
-            tol: Tolerance of the fitting problem, with account for variances in moments.
-                 Default value 1.95 corresponds to the two tail confidency 0.95.
-            reg_param: Regularization parameter.
+        :param quantity: mlmc.quanitity.Quantity instance, quantity for which the density is reconstructed
+        :param moments_fn: mlmc.moments
+        :param sample_storage: mlmc.sample_storage.SampleStorage instance, quantity data are stored there
+        :param tol: Tolerance of the fitting problem, with account for variances in moments.
+                    Default value 1.95 corresponds to the two tail confidence 0.95.
+        :param reg_param: regularization parameter
+        :return: None
         """
         distr_obj, result, _, _ = estimator.construct_density(tol=tol, reg_param=reg_param)
-        #distr_plot = mlmc.tool.plot.Distribution(title="{} levels, {} moments".format(self.n_levels, self.n_moments))
-        distr_plot = mlmc.tool.plot.ArticleDistribution(title="{} levels, {} moments".format(self.n_levels, self.n_moments))
+        # distr_plot = mlmc.tool.plot.Distribution(title="{} levels, {} moments".format(self.n_levels, self.n_moments))
+        distr_plot = mlmc.tool.plot.ArticleDistribution(
```

Comment:
Article specific stuff should rather be placed into a tagged branch out of the master. 
1. It allows various hacks without disturbing the master.
2. It keeps specific changes related to the reason (particular publication).

Reply:
OK, but I would do it manually at the end, because there are just few compact methods and class.